frontend/packages/dev-console/src/components/gitops/GitOpsServiceDetailsSection.scss

```diff
@@ -0,0 +1,23 @@
+.odc-gitops-service-details {
+  &__app-title {
+    font-weight: bold;
+    font-size: 18px;
+  }
+  &__app-pod {
+    width: 80px;
+    font-size: 12px;
+    padding: 10px 13px 10px 0px;
+  }
+  &__app-pr-details {
+    font-size: 12px;
+    padding-top: 20px;
```

Comment:
please use Pf variables wherever possible?

Reply:
I don't see anything aligning to PF variables value here. The font size could be `--pf-global--FontSize--xs` but as you pointed out in your other PR, there's discrepancies in the docs and actual values.